ext/opcache/tests/preload_user_002.phpt

```diff
@@ -0,0 +1,28 @@
+--TEST--
+preload_user sets the process uid
+--INI--
+opcache.enable=1
+opcache.enable_cli=1
+opcache.optimization_level=-1
+opcache.preload={PWD}/preload_user.inc
+opcache.preload_user={ENV:TEST_NON_ROOT_USER}
+opcache.log_verbosity_level=2
+--EXTENSIONS--
+opcache
+posix
+--SKIPIF--
+<?php
+if (PHP_OS_FAMILY == 'Windows') die('skip Preloading is not supported on Windows');
+if (posix_geteuid() !== 0) die('skip Test needs root user');
+?>
+--FILE--
+<?php
+var_dump(function_exists("f1"));
+var_dump(function_exists("f2"));
+?>
+OK
+--EXPECTF--
+bool(false)
```

Comment:
Hm, the expectation looks incorrect, given that `--FILE--` only has 2 `var_dump`s. I'm guessing this test is not actually executed in any job? AFAIK only Cirrus runs jobs as root.

Reply:
Ah, one of the `var_dump`s comes from the preloaded file. I was confused for a second :D